themes/openy_themes/openy_carnation/templates/page/page.html.twig

```diff
@@ -80,9 +80,9 @@
 
         {% if display_search %}
           <div class="search-form-wrapper col-12 border-top border-bottom">
-            <form method="get" action="{{ base_path }}/search/node">
+            <form method="get" action="{{ base_path }}/search">
               <i class="fa fa-search search-input-decoration" aria-hidden="true"></i>
-              <input type="search" name="keys" class="search-input" placeholder="Search">
+              <input type="search" name="{{ search_key }}" class="search-input" placeholder="{{ 'Search'|t }}">
```

Comment:
It seems the `search_key` variable is always empty, no matter if the module is enabled or not

Reply:
Fixed that